polybius.py

```diff
@@ -29,7 +30,7 @@ def get_one_time_pad(numbers):
 def dec_to_bin(x):
     #converts integers to decimal form
     start = '00'
-    return start + str(int(bin(x)[2:]))
+    return start + str(bin(x)[2:])
```

Comment:
Actually, I'm pretty sure bin() returns a string already, so you could just have bin(x)[2:]